zqd/ingest/archivepcap.go

```diff
@@ -62,18 +62,19 @@ func newArchivePcapOp(ctx context.Context, logstore storage.Storage, pcapstore *
 		return nil, warnings, err
 	}
 	p := &archivePcapOp{
+		importWriter: logstore.NewWriter(ctx),
```

Comment:
Nit: importWriter is a surprising name given that "import" doesn't appear anywhere else in this file. Maybe call this archiveWriter or logWriter?